### PR TITLE
Updated designs

### PR DESCRIPTION
**Pull request by MikaVohl**
### List of changes:

- Added new logo
- Updated favicon
- Added new FAQ design
- Added new images and removed boxes for About and Sponsor
- Added "and more coming soon" code and commented it out for sponsors
- Added more color blurs
- Made the MLH logo not clickable when it is invisible
- Made navbar links dependent on the width of the screen to avoid overlap

### Type of change:

- [x] Bug fix
- [x] New feature
- [ ] Breaking change
- [ ] New release
- [ ] Requires a documentation update

### CODE REVIEW

**Review by jackyyzhang03 — CHANGES_REQUESTED** [review]
On smaller screen sizes, faq content can overflow

Screen Shot 2023-12-05 at 12 09 04 PM

File: package.json [hunk 1]
```diff
@@ -42,7 +42,8 @@
     "react-visibility-sensor": "^5.1.1",
     "sass": "^1.49.0",
     "styled-components": "^4.4.0",
-    "swiper": "^8.4.2"
+    "swiper": "^8.4.2",
+    "use-resize-observer": "^9.1.0"
```
Comment (jackyyzhang03) [inline package.json]: Did we end up using this package?

**Comment by jackyyzhang03** [comment]
There appear to be some gaps that shouldn't be there

Screen Shot 2023-12-05 at 12 02 58 PM

File: package.json (hunk 1 above)
Comment (MikaVohl) [reply package.json]: No we didn't, I just uninstalled it right now for the next commit

**Review by jackyyzhang03 — APPROVED** [review]
LGTM

**Pull request merged**